quiz/util.py

```diff
@@ -6,3 +9,7 @@ def get_client_ip(request):
     else:
         ip = request.META.get('REMOTE_ADDR')
     return ip
+
+
+def get_published_questions():
```

Comment:
Maybe we'll find a better home for this eventually, but it can live here for now until a natural group of functions emerges that we can extract to somewhere more meaningful than `util`.

Reply:
Yeah, I think it's fine for now. But can create `model_queries.py`, `query_filters.py`, or something similar if you wish.

Reply:
Let's just extract it later if an obvious place appears.